terra_sdk/core/compact_bit_array.py

```diff
@@ -29,12 +29,6 @@ def to_data(self) -> dict:
             "elems": base64.b64encode(self.elems)
         }
 
-    def to_data(self) -> dict:
```

Comment:
@JSHan94 why is it removed?

Reply:
same function declared twice. see 26-31 lines